src/saagharapplication.cpp

```diff
@@ -236,8 +236,8 @@ void SaagharApplication::setupPaths()
         resourcesPath = QCoreApplication::applicationDirPath();
         dataPath = QDir::homePath() + "/Pozh/Saaghar/";
 #endif
-#ifdef Q_OS_X11
-        resourcesPath = PREFIX"/share/saaghar/";
+#ifdef Q_OS_LINUX
+        resourcesPath = "/usr/share/saaghar/";
```

Comment:
Thanks, but we don't need to change this line since PREFIX is defined in pro file. The real issue is that Q_OS_X11 is not defined on Arch. Also, I afraid that replacing Q_OS_X11 by Q_OS_X11 create the same bug on other OSes that Saaghar has been ported (e.g. FreeBSD). So, please replace it by: `#if defined(Q_OS_UNIX) && !defined(Q_OS_DARWIN)`

Reply:
You're right, now it has been updated !